### PR TITLE
Removed unused template, added modules list filter in SReccuringOrders

### PR DESCRIPTION
**Pull request by slawekk87**
## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/SRecurringOrders/models/Field.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	 */
+	public function getModulesListValues()
+	{
+		$modules = parent::getModulesListValues();
+		if ($this->getFieldName() !== 'target_module') {
+			return $modules;
+		}
+		foreach ($modules as $id => $module) {
+			if ($module['name'] !== 'SSingleOrders') {
+				unset($modules[$id]);
```
Comment (TomaszQr) [inline modules/SRecurringOrders/models/Field.php]: break;

File: modules/SRecurringOrders/models/Field.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Sławomir Kłos <s.klos@yetiforce.com>
+ */
+class SRecurringOrders_Field_Model extends Vtiger_Field_Model
+{
+	/**
+	 * Function to get all the available picklist values for the current field.
+	 *
```
Comment (TomaszQr) [inline modules/SRecurringOrders/models/Field.php]: /**
	 * {@inheritdoc}
	 */

File: modules/SRecurringOrders/models/Field.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	 * @return array List of picklist values if the field is of type picklist or multipicklist, null otherwise
+	 */
+	public function getModulesListValues()
+	{
+		$modules = parent::getModulesListValues();
+		if ($this->getFieldName() !== 'target_module') {
+			return $modules;
+		}
+		return $modules[\App\Module::getModuleId('SSingleOrders')];
```
Comment (TomaszQr) [inline modules/SRecurringOrders/models/Field.php]: ?

File: modules/SRecurringOrders/models/Field.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getModulesListValues()
+	{
+		if ($this->getFieldName() !== 'target_module') {
+			return parent::getModulesListValues();
+		}
+		return [App\Module::getModuleId('SSingleOrders') => ['name' => 'SSingleOrders', 'label' => \App\Language::translate('SSingleOrders', 'SSingleOrders')]];
```
Comment (rskrzypczak) [inline modules/SRecurringOrders/models/Field.php]: to many 'SSingleOrders'
add variable

**Comment by rskrzypczak** [comment]
Actually, the whole function should be in uitype.

**Pull request merged**